addons/survey/views/survey_templates.xml

```diff
@@ -330,115 +329,126 @@
+                <textarea type="text" class="form-control" t-att-name="'%s_%s' % (question.id, 'comment')" data-oe-survey-othert="1"><t t-esc="comment_line.value_text if comment_line else ''"/></textarea>
             </div>
             <div t-if='question.comments_allowed and not question.comment_count_as_answer' class="col-lg-12 mt16">
                 <span t-field="question.comments_message"/>
-                <textarea type="text" class="form-control" rows="1" t-att-name="'%s_%s' % (prefix, 'comment')"/>
+                <textarea type="text" class="form-control o_survey_comment" t-att-name="'%s_%s' % (question.id, 'comment')"><t t-esc="comment_line.value_text if comment_line else ''"/></textarea>
             </div>
         </div>
-        <div t-if="question.display_mode == 'columns' or not answer.token" class="row js_radio">
+        <div t-if="question.display_mode == 'columns'" class="row o_survey_form_simple_radio">
             <div t-foreach='question.labels_ids' t-as='label' t-attf-class="col-lg-#{question.column_nb}">
                 <label t-att-class="' bg-success ' if quizz_correction and label.answer_score > 0.0 else None">
-                    <input type="radio" t-att-name="prefix" t-att-value='label.id' />
+                    <input type="radio" t-att-name="question.id" t-att-value='label.id'
+                           t-att-checked="answer_line and answer_line.value_suggested.id == label.id and 'checked' or None"/>
                     <span t-field='label.value'/>
                 </label>
             </div>
             <div t-if='question.comments_allowed and question.comment_count_as_answer' class="js_comments col-lg-12" >
                 <label>
-                    <input type="radio" t-att-name="prefix" value="-1"/>
+                    <input type="radio" class="o_survey_form_radio_other" t-att-name="question.id" value="-1"
+                           t-att-checked="comment_line and 'checked' or None"/>
                     <span t-field="question.comments_message" />
                 </label>
-                <textarea type="text" class="form-control" rows="1" t-att-name="'%s_%s' % (prefix, 'comment')"/>
+                <textarea type="text" class="form-control" t-att-name="'%s_%s' % (question.id, 'comment')"><t t-esc="comment_line.value_text if comment_line else ''"/></textarea>
             </div>
             <div t-if='question.comments_allowed and not question.comment_count_as_answer' class="col-lg-12">
                 <span t-field="question.comments_message"/>
-                <textarea type="text" class="form-control" rows="1" t-att-name="'%s_%s' % (prefix, 'comment')" data-oe-survey-othert="1"/>
+                <textarea type="text" class="form-control o_survey_comment" t-att-name="'%s_%s' % (question.id, 'comment')"
+                          data-oe-survey-othert="1"><t t-esc="comment_line.value_text if comment_line else ''"/></textarea>
```

Comment:
What's this `data-oe-survey-othert` thing?

Reply:
don't know.. not used anyway so removed ! Thanks !